editor/export/project_export.cpp

```diff
@@ -1102,6 +1184,36 @@ ProjectExportDialog::ProjectExportDialog() {
 	include_margin->add_child(include_files);
 	include_files->connect("item_edited", callable_mp(this, &ProjectExportDialog::_tree_changed));
 	include_files->connect("check_propagated_to_item", callable_mp(this, &ProjectExportDialog::_check_propagated_to_item));
+	include_files->connect("custom_popup_edited", callable_mp(this, &ProjectExportDialog::_tree_popup_edited));
+
+	server_strip_message = memnew(Label);
+	server_strip_message->set_visible(false);
+	server_strip_message->set_autowrap_mode(TextServer::AUTOWRAP_WORD_SMART);
+	resources_vb->add_child(server_strip_message);
+
+	{
+		List<StringName> resource_names;
+		ClassDB::get_inheriters_from_class("Resource", &resource_names);
+
+		PackedStringArray strippable;
+		for (StringName resource_name : resource_names) {
+			if (ClassDB::has_method(resource_name, "create_placeholder", true)) {
+				strippable.push_back(resource_name);
+			}
+		}
+		strippable.sort();
+
+		String message = TTR("Strip visuals will replace the following resources with placeholders: ");
```

Comment:
Use quotes to refer to option names:

```suggestion
		String message = TTR("\"Strip Visuals\" will replace the following resources with placeholders: ");
```